charter.html

```diff
@@ -262,6 +271,11 @@ <h2>Success Criteria</h2>
           <p>Normative specifications which have user-facing features should contain a section on accessibility that describes the benefits and impacts, including ways specification features can be used to address them, and recommendations for maximizing accessibility in implementations.</p>
 	</section>
 
+  <h5 id="success-criteria-notes">[2] Note on Success Criteria</h5>
```

Comment:
Is this section intended to be included @jwrosewell ?

Reply:
Not in the final version.